index.js

```diff
@@ -1,16 +1,29 @@
 const PullRequest = require('./src/github').PullRequest;
 
 module.exports = app => {
-  app.on(['pull_request.opened'], pullRequest)
+  app.on(['pull_request.opened', 'pull_request.synchronized'], onPullRequest)
+  app.on(['check_run.rerequested'], onCheckRunRerequest)
```

Comment:
Cool! I assume this is the event that gets triggered when someone clicks on the "re-run" link?

Reply:
that is correct!